Modules/_operator.c

```diff
@@ -731,9 +731,9 @@ _operator_is_not_impl(PyObject *module, PyObject *a, PyObject *b)
 /*
  * timing safe compare
  *
- * Returns 1 of the strings are equal.
+ * Returns 1 if the strings are equal.
  * In case of len(a) != len(b) the function tries to keep the timing
- * dependent on the length of b. CPU cache locally may still alter timing
+ * dependent on the length of b. CPU cache locality may still alter timing
```

Comment:
The grammar in this sentence isn't ideal, but I'm not sure this proposed change reflects the intended meaning of the comment. I think this change would make more sense:

```suggestion
 * dependent on the length of b. The CPU cache locally may still alter timing
```

Reply:
@kumaraditya303 @DavidBuchanan314  — if I'm misunderstanding what the comment is trying to say, please correct me :)

Reply:
"cache locality" is domain-specific jargon with special meaning. Your adjustment changes the meaning. I can't find a good dictionary-definition for it, but if you don't believe me, you can see plenty of usages of it on stackoverflow: https://stackoverflow.com/search?q=cache+locality

Reply:
@AlexWaygood "cache locality" is the correct term which at a high level means that the data will be present in CPU cache and the operation will be faster (no round trips to RAM).

Reply:
It's also a term used elsewhere in the cpython source tree, e.g. https://github.com/python/cpython/blob/9833bb91e4d5c2606421d9ec2085f5c2dfb6f72c/Objects/dictnotes.txt#L120 (in most contexts, such as that one, cache locality is a good thing since it improves performance - but in the case of constant-time comparisons, it's undesirable due to potentially data-dependant timing - hence the comment)

Reply:
Alrighty, thanks both — TIL! Please excuse my ignorance here :)